test/rule-matches/color-contrast-matches.js

```diff
@@ -537,4 +537,32 @@ describe('color-contrast-matches', function () {
       assert.isTrue(rule.matches(input, axe.utils.getNodeFromTree(input)));
     });
   }
+
+  describe('with font icons', () => {
```

Comment:
Needs a true case

Reply:
The way I figured it, pretty much every other test in this suite shows that text that isn't an icon font can pass or fail the check. Whatever I add here would basically just be a copy of the default "true" case.